packages/victory-bar/src/bar.js

```diff
@@ -118,7 +120,9 @@ const Bar = (props) => {
     shapeRendering: props.shapeRendering,
     clipPath: props.clipPath,
     desc: props.desc,
-    tabIndex: props.tabIndex
+    tabIndex: props.tabIndex,
+    "aria-label": props.ariaLabel,
+    "aria-describedby": props.ariaDescribedBy
```

Comment:
I don't see `aria-description` included in the latest wai-aria spec. Could you point me towards the documentation for it? Also, what would be the benefit of using `aria-labelledby` over `aria-label` in the case of charts?

Reply:
Oh sorry you're correct - `aria-description` isn't real. `aria-labelledby` would allow you to do something like tell a screen reader that an element of a chart is labelled by a particular element in a legend as opposed to duplicating the label on both elements.